src/PlusDataCollection/WinProbe/vtkPlusWinProbeVideoSource.cxx

```diff
@@ -1762,20 +1762,31 @@ int32_t vtkPlusWinProbeVideoSource::GetARFIStopSample()
 }
 
 //----------------------------------------------------------------------------
-PlusStatus vtkPlusWinProbeVideoSource::SetARFIPushConfigurationString(std::string pushConfiguration)
+void vtkPlusWinProbeVideoSource::SetARFIPushConfigurationString(std::string pushConfiguration)
 {
   if(Connected)
   {
     m_ARFIPushConfigurationCount = std::count(pushConfiguration.begin(), pushConfiguration.end(), ';') + 1;
-    if(quadBFCount == 1 && m_ARFIPushConfigurationCount != 30)
+    if(quadBFCount == 1)
     {
-      LOG_ERROR("A X4BF requires an ARFI Push Configuration string to have 30 pushes so that the buffers are sized correctly.");
-      return PLUS_FAIL;
+      if(m_ARFIPushConfigurationCount == 30)
+      {
+        LOG_WARNING("A X4BF will use an API hardcoded 30 push configuration rather than the custom specified configuration that was set.")
+      }
+      else
+      {
+        LOG_ERROR("A X4BF requires an ARFI Push Configuration string to have 30 pushes so that the buffers are sized correctly. "
+                  "An appropriate configuration length will be set instead.");
+        pushConfiguration = "1,40,48;1,48,56;1,56,64;1,64,72;1,72,80;1,80,88;"
```

Comment:
This variable is assigned to, but not used. As it is a parameter that is passed by value, this will not be seen from outside this method.

Reply:
As in it should be the following at time of set? Currently `m_ARFIPushConfigurationString` is only updated at time of `GetARFIPushConfigurationString`.
```C++
m_ARFIPushConfigurationString = pushConfiguration
WPSetARFIPushConfigurationString(pushConfiguration.c_str());
```

Reply:
I didn't see it being used on line 1787. Sorry for the noise.